src/Bean_A.las

```diff
@@ -0,0 +1,10431 @@
+~Version Information Block

```

Comment:
Do not commit a data file to the repository. We will be using DataDeps.jl to write tests and download the data dynamically.

Reply:
Also, I think we now need to start a fresh PR to avoid this big file commit. Git becomes slow when we commit large files.

Reply:
That's why I temporarily added *.las in the gitignore. Will delete the file in this commit